torch/onnx/_internal/jit_utils.py

```diff
@@ -98,6 +99,49 @@ def aten_op(self, operator: str, *args, overload_name: str = "", **kwargs):
             **kwargs,
         )
 
+    @_beartype.beartype
+    def onnxscript_op(
+        self,
+        onnx_fn,  # TODO(titaiwang): annotate this when onnx-script becomes dependency
+        *raw_args: Union[torch.Tensor, _C.Value],
+        outputs: int = 1,
+        **kwargs,
+    ):
+        """Creates an ONNX operator from onnx-script function, taking "raw_args" as inputs and "kwargs" as attributes.
```

Comment:
I am unclear what is the intended usage here? I thought this was for static-graph building? Can it also be used in a dynamic setting? It's probably just my lack of familiarity with GraphContext and its usage. I was wondering about the use of kwargs/args, but I guess this is just inheriting/reusing the usage-pattern of the existing op/_add_op methods, so maybe it doesn't matter.